lib/monads/monads.mli

```diff
@@ -726,9 +726,9 @@ module Std : sig
             evaluation is unspecified.  *)
         val all : ('a,'e) m t -> ('a t, 'e) m
 
-        (** [all_ignore cs] performs all computations in [cs] in an
+        (** [all_unit cs] performs all computations in [cs] in an
             unspecified order, and discards the results. *)
-        val all_ignore : ('a,'e) m t -> (unit,'e) m
+        val all_unit : ('a,'e) m t -> (unit,'e) m
```

Comment:
we need to preserve the old interface, otherwise we need to release Monads 2.0. 
I don't like the idea of bumping the major version just because of changes in Core.